app/styles/components/_grid.scss

```diff
@@ -1,11 +1,26 @@
 $grid-max-width: 1200px;
+$grid-break-mobile: 1049px;
 
 .Grid {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  justify-content: space-between;
+  justify-content: center;
 
   max-width: $grid-max-width;
   margin-top: $Theme-margin-half;
 }
+
+.Grid--marginRightDesktop {
+  margin-right: $Theme-margin-default;
+}
+
+@media (max-width: $grid-break-mobile) {
+  .Grid {
+    flex-direction: column;
+  }
+  
+  .Grid--marginRightDesktop {
+    margin-right: 0;
+  }
+}
```

Comment:
I used this solution because I checked with @fcbaila and he confirmed that Grid will remain two-columned, with new people for both Board or Officers being added to a new row instead of a third column.